extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -90,14 +93,35 @@ amp-carousel .amp-carousel-button.amp-disabled {
 
 .-amp-slide-item {
   display: none;
+  flex: 1 0 100% !important;
   height: 100% !important;
   position: relative;
-  scroll-snap-coordinate: 0 0;
+  scroll-snap-coordinate: 0 0 !important;
   width: 100% !important;
 }
 
 .-amp-slide-item-show {
-  display: inline-block !important;
+  display: block !important;
```

Comment:
Ok. Let's try it out later. The benefit would be that the slide's contents are centered.